skimage/measure/__init__.py

```diff
@@ -7,7 +7,7 @@
 from ._pnpoly import points_in_poly, grid_points_in_poly
 from ._moments import moments, moments_central, moments_normalized, moments_hu
 from .profile import profile_line
-from .fit import LineModel, CircleModel, EllipseModel, ransac
+from .fit import LineModel, LineModel3D, CircleModel, EllipseModel, ransac
```

Comment:
Hi,

I am not that keen on this idea.

It would be easy to do something like:

    def LineModel(BaseModel):

        def estimate(self, data):

	    if data.shape[1] == 2:
		    self._model = LineModel2D
	    else:
		    self._model = LineModel3D
		
	    self._model.estimate(data)
	    self.params = self._model.params

However, I do not like the fact that LineModel2D has predict_x() and predict_y() functions, which LineModel3D has not. So combining them becomes awkward. Also, one model stores a line as (offset, theta) while the other stores (point, unit vector). So the unique LineModel should specify in its documentation that `params` do not have the same meaning in 2D and in 3D. Hence the rest of the code cannot be unaware of the fact it is written either for 2D or 3D.

While I agree in calling LineModel2D the current LineModel, I do not agree on making a single LineModel that would have different behaviours in 2D or 3D.